src/main/play-doc/operation/Install.md

```diff
@@ -191,43 +191,56 @@ On some Debian distributions you may need to add a dedicated Personal Package Ar
+ConductR provides ConductR-HAProxy bundle that listens for bundle events from ConductR and updates HAProxy configuration accordingly. 
 
-``` bash
-[172.17.0.1]$ sudo dpkg -i /usr/share/conductr/extra/conductr-haproxy_%PLAY_VERSION%_all.deb
-```
-or
+Grant HAProxy configuration file read and write access to ConductR-HAProxy bundle:
 
 ``` bash
-[172.17.0.1]$ sudo yum install  /usr/share/conductr/extra/conductr-haproxy-%PLAY_VERSION%-1.noarch.rpm
+[172.17.0.1]$ sudo chown conductr:conductr /etc/haproxy/haproxy.cfg
 ```
 
-Grant HAProxy configuration file read and write access to ConductR-HAProxy application:
+After updating the configuration file ConductR-HAProxy is going to signal HAProxy to reload its configuration. Grant permissions for the particular command that ConductR-HAProxy is going to use by modifying the `sudoers` file:
 
 ``` bash
-[172.17.0.1]$ sudo chown conductr-haproxy:conductr-haproxy /etc/haproxy/haproxy.cfg
+[172.17.0.1]$ echo "conductr ALL=(root) NOPASSWD: /etc/init.d/haproxy reload" | sudo tee -a /etc/sudoers
 ```
 
-After updating the configuration file ConductR-HAProxy is going to signal HAProxy to reload its configuration. Grant permissions for the particular command that ConductR-HAProxy is going to use by modifying the `sudoers` file:
+On RHEL and CentOS it may also be neccessary to [disable default requiretty](https://bugzilla.redhat.com/show_bug.cgi?id=1020147) for the conductr user.
 
 ``` bash
-[172.17.0.1]$ echo "conductr-haproxy ALL=(root) NOPASSWD: /etc/init.d/haproxy reload" | sudo tee -a /etc/sudoers
+[172.17.0.1]$ echo 'Defaults: conductr  !requiretty' | sudo tee -a /etc/sudoers
 ```
 
-On RHEL and CentOS it may also be neccessary to [disable default requiretty](https://bugzilla.redhat.com/show_bug.cgi?id=1020147) for the conductr-haproxy user.
+## Loading and Running ConductR-HAProxy Bundle
 
-``` bash
-[172.17.0.1]$ echo 'Defaults: conductr-haproxy  !requiretty' | sudo tee -a /etc/sudoers
-```
+ConductR-HAProxy bundle listens for bundle changes within ConductR and updates the HAProxy config to expose the bundle endpoints accordingly.
 
-Set the ConductR IP address which is going to be used by ConductR-HAProxy to listen to bundle events in the cluster:
+### Prepare ConductR-HAProxy nodes
 
-``` bash
-[172.17.0.1]$ echo -Dconductr-haproxy.ip=$(hostname)| sudo tee -a /usr/share/conductr-haproxy/conf/application.ini
-[172.17.0.1]$ sudo service conductr-haproxy restart
+_Repeat each step in this section also on the `172.17.0.2` and `172.17.0.3` machine._
+
+ConductR-HAProxy bundle needs to be installed where HAProxy is installed, and these nodes can be distinguished by the `haproxy` role. Assign the `haproxy` role to the nodes where the proxy will be hosted.
```

Comment:
'needs to be installed where HAProxy is installed' 
Can this be clearer/more prescriptive? 'must be installed on all nodes where HAProxy is installed'